configs/stable_diffusion/base_config_denoisingunet.py

```diff
@@ -0,0 +1,87 @@
+unet = dict(
+    type='DenoisingUnet',
+    image_size=512,
+    base_channels=320,
+    channels_cfg=[1, 2, 4, 4],
+    unet_type='stable',
+    act_cfg=dict(type='silu', inplace=False),
```

Comment:
Seems only `type` is used. I think we can remove `inplace`.

Reply:
resolved.